provision/acc_provision/templates/provision-config.yaml

```diff
@@ -1,6 +1,8 @@
 #
 # Configuration for ACI Fabric
 #
+operator_managed_config: 
+  enable_updates: false         
```

Comment:
This file is an example to the user as to how to change the default configuration, so we don't provide the default configuration (because there is already in the system). The above should be:

      #
      # Configuration for ACI CNI Operator
      #

      #
      # operator_managed_config:
      #   enable_updates: True    # Default is False

Also, this is not "ACI Config", so please move this to L86 (after the kube_config and before istio_config sections).